Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeHost.mm

```diff
@@ -501,7 +492,8 @@ static void recursivelyMapIOSurfaceBackingStore(CALayer *layer)
 void RemoteLayerTreeHost::remotePageProcessCrashed(WebCore::ProcessIdentifier processIdentifier)
 {
     for (auto layerID : m_hostedLayersInProcess.take(processIdentifier)) {
-        [layerForID(layerID) removeFromSuperlayer];
+        if (auto* node = nodeForID(layerID))
+            node->setHostingNode(nullptr);
```

Comment:
I'd call this a different name, like "removeFromHostingNode", then rename setHostingNode to "addToHostingNode" and make it take a RemoteLayerTreeNode&